exercises/octal/src/test/java/OctalTest.java

```diff
@@ -1,49 +1,99 @@
 import org.junit.Test;
 import org.junit.Ignore;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-import java.util.Arrays;
-import java.util.Collection;
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(Parameterized.class)
 public class OctalTest {
+    private Octal octal;
```

Comment:
This doesn't need to be a private variable since it's being assigned to a new object in each test. It would be just as easy (and would save you one little line of code 😛 ) to remove the private variable and change each test to have:
```
Octal octal = new Octal("1");
```
🙂

Reply:
I checked some of the other exercise tests and the ones I checked have the same private variable declaration at the beginning as well. For consistency should it be looked at to change those as well to match if it is changed here?

Reply:
+1 to what @c-thornton said. I looked at some other tests and found the same thing, so I followed what was done. I've made changes to declaring a new variable in each test case. In light of the inconsistency for the test cases, is one way of declaration always preferred over the other?

Reply:
It could be something worth looking at :slightly_smiling_face: 

I'll do some investigating later and see what the situation is like!

Reply:
That does seem worth looking at, thanks for raising it @c-thornton and @Haozhe321! 🙂 

I think it's good practice to have a private variable (initialised in a `@Before` method) if the class constructor takes no arguments. In that case it's useful since it saves you from having to repeat the line initialising the variable in each test.
If the constructor does take arguments then it would have to be initalised in each test in which case I don't think it's worth having a private variable. Does that sound sensible? 🙂